examples/dask-nvtabular-criteo-benchmark.py

```diff
@@ -38,6 +40,39 @@ def setup_rmm_pool(client, pool_size):
     return None
 
 
+def setup_dirs(BASE_DIR, dask_workdir, output_path, stats_path):
+    # GCP Storage
+    if "gs://" in BASE_DIR:
+        # Get client and bucket
+        storage_client = storage.Client()
```

Comment:
Here, we should raise an error if `google.cloud.storage` is not installed (and tell the user they need that library to handle `"gs://"`.  Same idea for aws below.